Misc/NEWS.d/next/Build/2019-10-13-16-18-24.bpo-38468.buCO84.rst

```diff
@@ -0,0 +1,4 @@
+Use `getvar` for all still existing `sysconfig.get_config_var()` calls.
+The affected file is `Python3.9-config.in`.
+Misc/python-config.in now uses `getvar` for all still existing `sysconfig.get_config_var()` calls.
+Patch by Joannah Nanjekye 

```

Comment:
Something gone wrong :-/ I only expected 2 lines for this file, not 4. Maybe I shouldn't use GitHub "suggestion" on multiple lines :-( I expected:

```
Misc/python-config.in now uses `getvar()` for all still existing `sysconfig.get_config_var()` calls.
Patch by Joannah Nanjekye.
```